packages/flutter/lib/src/painting/debug.dart

```diff
@@ -31,3 +33,19 @@ bool debugAssertAllPaintingVarsUnset(String reason, { bool debugDisableShadowsOv
   }());
   return true;
 }
+
+/// Signature for a method that returns an [HttpClient].
```

Comment:
Usually the `debugAssertAll*VarsUnset` method is the last one in a `debug.dart` file. You may want to move these newly introduced members up above it.

Reply:
done